posthog/migrations/0195_alter_person_created_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.5 on 2022-01-04 14:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
This migration is likely to cause an outage I think due to locking the table. Also I don't think we need it to solve this task.

Reply:
This is just modifying `auto_add_not` on the `models.DateTimeField` and generates the following SQL:

```
BEGIN;
--
-- Alter field created_at on person
--
COMMIT;
```

i.e. it doesn't do anything.

Anyway, if we can avoid updating auto_add_now I would much prefer!

Reply:
Ah, if we're confident it won't cause issues then :+1: let's do this.

Not quite sure what the issue is/. Would dropping postgres support for the query simplify anything? (We can do so easily)